### PR TITLE
feat: converters for `ngrx` rules

### LINKED ISSUE #1056: Add missing ngrx rule converters
#### Existing Behavior

There is another TSLint plugin that needs converting to the existing ESLint equivalent for the ngrx library:

- [ngrx-tslint-rules](https://github.com/timdeschryver/ngrx-tslint-rules)
- [eslint-plugin-ngrx](https://github.com/timdeschryver/eslint-plugin-ngrx)

We can use the [migration guide](https://github.com/timdeschryver/eslint-plugin-ngrx/blob/main/docs/migrating-from-ngrx-tslint-rules.md) provided at the eslint plugin docs.

#### Change Proposal

Add the missing converters. Should be easy to setup after adding #813 feature as most rules do not require configuration.

### PR DESCRIPTION
**Pull request by rafaelss95**
## PR Checklist

-   [x] Addresses an existing issue: fixes #1056
-   [x] That issue was marked as [`status: accepting prs`](https://github.com/typescript-eslint/tslint-to-eslint-config/labels/status%3A%20accepting%20prs)

## Overview
This PR adds converters for `ngrx-*` rules according to the [migration guide](https://github.com/timdeschryver/eslint-plugin-ngrx/blob/main/docs/migrating-from-ngrx-tslint-rules.md).
The only missing rule is `ngrx-no-duplicate-action-types`, which was [not migrated](https://github.com/timdeschryver/eslint-plugin-ngrx/blob/main/docs/migrating-from-ngrx-tslint-rules.md#not-migrated).

### CODE REVIEW

File: src/converters/lintConfigs/rules/ruleConverters.ts [hunk 1]
```diff
@@ -64,6 +64,17 @@ import { convertCurly } from "./ruleConverters/curly";
 import { convertCyclomaticComplexity } from "./ruleConverters/cyclomatic-complexity";
 import { convertDeprecation } from "./ruleConverters/deprecation";
 import { convertEofline } from "./ruleConverters/eofline";
+import { convertActionHygiene } from "./ruleConverters/eslint-plugin-ngrx/action-hygiene";
```
Comment (rafaelss95) [inline src/converters/lintConfigs/rules/ruleConverters.ts:67]: I think the folder name would be `ngrx-tslint-rules` as, like `codelyzer`, it refers to the **tslint** package, but as there are several other folders referencing the name of **eslint** package, I was confused and I preferred to follow the majority 🤷‍♂️

Comment (JoshuaKGoldberg) [reply src/converters/lintConfigs/rules/ruleConverters.ts:67]: Yeah good point. Filed a followup to take another look at these: #1197

File: src/converters/lintConfigs/rules/ruleConverters/eslint-plugin-ngrx/effect-creator-and-decorator.ts [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import { RuleConverter } from "../../ruleConverter";
+
+export const convertEffectCreatorAndDecorator: RuleConverter = () => {
+    return {
+        rules: [
+            {
+                ruleName: "ngrx/effect-decorator-and-creator",
```
Comment (JoshuaKGoldberg) [inline src/converters/lintConfigs/rules/ruleConverters/eslint-plugin-ngrx/effect-creator-and-decorator.ts:7]:
```suggestion
                ruleName: "ngrx/no-effect-decorator-and-creator",
```

**Review by JoshuaKGoldberg — CHANGES_REQUESTED** [review]
Looks great to me, thanks! Just the one correction for `(no)-effect-decorator-and-creator`.

For `ngrx-no-duplicate-action-types`, the right move would be to make a converter that has no output rules. That way  tslint-to-eslint-config won't log a complaint about not knowing an equivalent output rule.

File: src/converters/lintConfigs/rules/ruleConverters/eslint-plugin-ngrx/effect-creator-and-decorator.ts (hunk 2 above)
Comment (rafaelss95) [reply src/converters/lintConfigs/rules/ruleConverters/eslint-plugin-ngrx/effect-creator-and-decorator.ts:7]: Good catch!

**Comment by rafaelss95** [comment]
> For `ngrx-no-duplicate-action-types`, the right move would be to make a converter that has no output rules. That way tslint-to-eslint-config won't log a complaint about not knowing an equivalent output rule.

Oh, thanks for clarifying this. In that case, I'll submit new PRs for the missing converters for `codelyzer` and `tslint-rxjs-rules` too.

File: src/converters/lintConfigs/rules/ruleConverters.ts (hunk 1 above)
Comment (rafaelss95) [reply src/converters/lintConfigs/rules/ruleConverters.ts:67]: Nice! Do you want me to update the folder name to `ngrx-tslint-rules` then?

Comment (JoshuaKGoldberg) [reply src/converters/lintConfigs/rules/ruleConverters.ts:67]: Sure, that'd be swell. Yes please!

**Review by JoshuaKGoldberg — APPROVED** [review]
Super, thanks (again) @rafaelss95!

**Comment by JoshuaKGoldberg** [comment]
I'll wait a bit in case you're still working on this (e.g. the folder renames), then merge before I go to bed tonight. Whatever you don't get in this PR I'd be happy to review as a fast follow.

**Comment by rafaelss95** [comment]
Done, I've renamed the folder to `ngrx-tslint-rules`.

**Comment by JoshuaKGoldberg** [comment]
Well, super - I'll go ahead and merge now!

**Pull request merged**